frontend/src/App.jsx

```diff
@@ -1,20 +1,34 @@
-import React from 'react';
-import axios from 'axios';
-import PhotoContainer from './components/photos/PhotoContainer';
+import React, { useEffect } from 'react';
+import Modal from './components/modal/Modal';
+import Photos from './components/photos/Photos';
+import UseWindowSize from './helpers/UseWindowSize';
+import MobileModal from './components/modal/mobile/MobileModal';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  getPhotoState,
+  handleGetPhotos,
+} from './redux/slices/photos/PhotoSlice';
 
 const App = () => {
-  const handleClick = async () => {
-    try {
-      const res = await axios.get('http://localhost:5000/api');
-      console.log(res);
-    } catch (error) {
-      console.log(error);
-    }
-  };
+  const dispatch = useDispatch();
+  const { width, height } = UseWindowSize();
+  const { current, allPhotos } = useSelector(getPhotoState);
+
+  useEffect(() => {
+    dispatch(handleGetPhotos());
+  }, []);
+
   return (
-    <div style={{ marginTop: '66px' }}>
-      <PhotoContainer />
-    </div>
+    current && (
+      <React.Fragment>
+        {width > 1127 ? (
```

Comment:
@manedurphy Curious, what is the purpose of the "width > 1127" conditional, is this for responsiveness? It is not immediately clear to me what this does.